architecture/service_catalog/index.adoc

```diff
@@ -101,6 +101,24 @@ references to the binding secret from the deployment. Otherwise, the next
 rollout will fail.
 ====
 
+[[service-catalog-route-planning]]
+== Service catalog compatibility with services outside of the cluster
+
+As of {product-title} version 3.10, a user can specify a Kubernetes service
```

Comment:
I think you are thinking of the wrong thing.

The service catalog allows you to provision things in your cluster, but it can be a linkage to something outside the cluster too.  If it's outside the cluster then egress rules apply, but they can't be set by anyone other then an admin, so it might be worth mentioning as a note at the end... but not as the main lead-in.  The point of this is that the internal self-hosted services will need network policy rules to be set or people can't access them.

Reply:
Given that this is in the service catalog section, I'm not sure how much of this preamble you need.